### ISSUE TITLE
When Amp Enhancer Popup is closed it z-index is still remained which is affecting other popup to get the control

### ISSUE THREAD

**Issue by kupveja** [ahmedkaludi/amp-enhancer#53]
When Amp Enhancer Popup is closed it z-index is still remained which is affecting other popup to get the control

help scout link - https://secure.helpscout.net/conversation/1504210242/194821?folderId=1060556

**Comment by Enricobt** [comment]
Hi
I am adding here some additional information about this problem that I discovered on my WP site.
--
I found a conflict between the Iubenda Plugin and POPUP in my AMP site version.
Iubenda plugin is used to Enable GDPR Cookie Consent and inform users about cookies and to accept them. Usually, the Iubenda acceptance windows come first and then after the user clicks on ACCEPT button, he can continue browsing the site.
With MP Enhancer Plugin, my own POPUP come first and then after closing it, user will see the IUBENDA pop for cookie policy. At that time the two buttons, Accept/Reject, are not clickable as there is a transparent area covering all.
This is coming from this class definition:
.i-amphtml-consent-ui-mask {
......
z-index: 2147483644;
}
I tried to insert in CSS the following instruction to decrease z-index and to make all buttons active, but unfortunately, this is not working and the z-index remains as it is.
.i-amphtml-consent-ui-mask {
z-index: 1;
}
